src/lib.rs

```diff
@@ -174,19 +184,19 @@ pub fn consumer(
                             new_results
                         }
                     }
-                    ItemType::Buffers(buffers) => {
+                    ItemType::Buffers(mut buffers) => {
                         // LLVM
-                        let result = call_parse_llvm_gcno_buf(
-                            working_dir.to_str().unwrap(),
-                            &buffers.stem,
-                            &buffers.gcno_buf,
-                            &buffers.gcda_buf,
-                            branch_enabled,
-                        );
-
-                        drop(buffers.gcda_buf);
-                        drop(buffers.gcno_buf);
-                        result
+                        match GCNO::compute(&buffers.stem,
+                                            buffers.gcno_buf,
+                                            buffers.gcda_buf,
+                                            branch_enabled) {
+                            Ok(r) => r,
+                            Err(e) => {
+                                // Just print the error, don't panic and continue
+                                eprintln!("{}", e);
```

Comment:
Add an error message here before `{}`, so when it happens we can tell where it comes from